pages/common/ansible-galaxy.md

```diff
@@ -1,6 +1,7 @@
 # ansible-galaxy
 
 > Create and manage Ansible roles.
+> Homepage: <https://docs.ansible.com/ansible/latest/reference_appendices/galaxy.html#the-command-line-tool>.
```

Comment:
Why not https://docs.ansible.com/ansible/latest/cli/ansible-galaxy.html, to match what you used for the `ansible-playbook` page?

I understand that the reference page is more informative, but I'm not sure it justifies the much harder to read URL. But in any case, we should use a consistent URL format for similar pages.

Reply:
Didn't notice! The Ansible documentation is rather confusing :P

I'll change it now.